bedrock/stories/templates/stories/stories-base-article.html

```diff
@@ -116,7 +116,7 @@ <h2 class="c-stories-article-subtitle">{% block article_subtitle %}{% endblock %
       </div>
       <section class="c-stories-social-share">
         <ul class="social-share-list">
-          <li class="social-share-item"><a target="_blank" href="https://twitter.com/share?url=http://www.mozilla.org{{ request.path }}&text={{ self.article_title()|urlencode  }}" class="twitter">Twitter</a></li>
+          <li class="social-share-item"><a target="_blank" href="https://twitter.com/share?url=http://www.mozilla.org{{ request.path }}&text={{ self.article_title()|urlencode  }}" class="twitter" aria-label="X (formerly Twitter)">X</a></li>
```

Comment:
I still see the bird icon here? (we can just use the link text instead of `aria-label` again here too?)